certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1,1411 +1,1422 @@
+                        "type": "string",
+                        "description": "lookupText"
+                      },
+                      "entityId": {
+                        "type": "string",
+                        "description": "entityId"
+                      },
+                      "lookupUrl": {
+                        "type": "string",
+                        "description": "lookupUrl"
+                      },
+                      "associatedOrgs": {
+                        "type": "array",
+                        "items": {
+                          "type": "object",
+                          "properties": {
+                            "name": {
+                              "type": "string",
+                              "description": "name"
+                            },
+                            "entityId": {
+                              "type": "string",
+                              "description": "entityId"
+                            },
+                            "lookupUrl": {
+                              "type": "string",
+                              "description": "lookupUrl"
+                            }
+                          }
+                        },
+                        "description": "associatedOrgs"
+                      }
+                    }
+                  },
+                  "description": "entities"
+                }
+              }
+            }
+          }
+        },
+        "summary": "NER - Entities",
+        "operationId": "ner-entities",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "object",
+                  "properties": {
+                    "enableIdLookup": {
+                      "type": "boolean",
+                      "description": "enableIdLookup"
+                    },
+                    "filterEntities": {
+                      "type": "boolean",
+                      "description": "filterEntities"
+                    },
+                    "text": {
+                      "type": "string",
+                      "description": "text"
+                    }
+                  },
+                  "description": "data"
+                }
+              }
+            }
+          }
+        ],
+        "description": "Extract named entities from document text."
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://auth.factset.com/as/authorization.oauth2",
+      "tokenUrl": "https://auth.factset.com/as/token.oauth2",
+      "scopes": {}
+    }
+  },
+  "security": [
+    {
+      "oauth2_auth": []
+    }
+  ],
+  "tags": []
```

Comment:
@KyleVErwin 'x-ms-connector-metadata' property is required. Please see out templates [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json#:~:text=%22x%2Dms%2Dconnector,%5D).